Formula/docker-credential-helper-ecr.rb

```diff
@@ -0,0 +1,25 @@
+class DockerCredentialHelperEcr < Formula
+  desc "Docker Credential Helper for Amazon ECR"
```

Comment:
The description shouldn't really just be the name.

Reply:
@ilovezfs I'm open to suggestions, but considering the character limit that `brew audit` enforces, there isn't any other good option.